testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java

```diff
@@ -208,7 +208,7 @@ public void run() {
             }
         };
         try {
-            executor.submit(r).get(1, TimeUnit.MINUTES);
+            executor.submit(r).get(2, TimeUnit.MINUTES);
```

Comment:
Hello @lvydra , would you mind using [TimeoutUtil.java](https://github.com/wildfly/wildfly-core/blob/main/testsuite/shared/src/main/java/org/jboss/as/test/shared/TimeoutUtil.java) for handling this timeout?

This utility class allows timeouts to be modified with a correction factor. Slow CI environments can append the `ts.timeout.factor` property to increase the default timeout, making the testsuite more reliable in slow environments.

Thanks